<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeuralNetworkWidgetBase</class>
 <widget class="QWidget" name="NeuralNetworkWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>77</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" columnstretch="2,1,0">
     <item row="6" column="1" colspan="2">
      <widget class="QLineEdit" name="leValidationError">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Momentum</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Learning Rate</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Neighbourhood</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Min Validation Overall Error</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Hidden Layers</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QSpinBox" name="spnMaxIterations">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>500</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QSpinBox" name="spnNeigbourhood">
       <property name="suffix">
        <string> px</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QDoubleSpinBox" name="spnMomentum">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>0.001000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.005000000000000</double>
       </property>
       <property name="value">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Maximum Iterations</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QLineEdit" name="leDeltaRMS">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="leTopology"/>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QDoubleSpinBox" name="spnLearnRate">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>0.001000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.005000000000000</double>
       </property>
       <property name="value">
        <double>0.020000000000000</double>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="leKappa">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Current Validation Kappa</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Î” Overall Accuracy</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QPushButton" name="btnStop">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QPushButton" name="btnTrainNetwork">
       <property name="text">
        <string>Train neural network</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="2">
    <layout class="QVBoxLayout" name="layoutPlot"/>
   </item>
  </layout>
  <zorder>verticalSpacer</zorder>
  <zorder></zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
